cmd/runmqserver/webserver.go

```diff
@@ -33,7 +35,22 @@ func startWebServer() error {
 		return nil
 	}
 	log.Println("Starting web server")
-	out, rc, err := command.RunAsMQM("strmqweb")
+	cmd := exec.Command("strmqweb")
+	// Set a default app password for the web server, if one isn't already set
+	_, set := os.LookupEnv("MQ_APP_PASSWORD")
+	log.Println(cmd.Env)
```

Comment:
Should we be printing out the env given that we are storing passwords in the env? Could result in passwords being transmitted to any container log scrapping services.